Gopkg.lock

```diff
@@ -1,585 +0,0 @@
-# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
I think we don't need to update lock file here can you remove this lock file from this PR?